src/components/Nav/index.tsx

```diff
@@ -102,83 +63,26 @@ const Nav = () => {
-                    transition="transform 0.5s, color 0.2s"
-                    icon={ThemeIcon}
-                    aria-label={themeIconAriaLabel}
-                    variant="ghost"
-                    isSecondary
-                    px={{ base: "2", xl: "3" }}
-                    _hover={{
-                      transform: "rotate(10deg)",
-                      color: "primary.hover",
-                    }}
-                    onClick={toggleColorMode}
-                  />
-
-                  {/* Locale-picker menu */}
-                  <LanguagePicker
-                    placement="bottom-end"
-                    minH="unset"
-                    maxH="75vh"
-                    w="xs"
-                    inset="unset"
-                    top="unset"
-                    menuState={languagePickerState}
-                  >
-                    <MenuButton
-                      as={Button}
-                      name={DESKTOP_LANGUAGE_BUTTON_NAME}
-                      ref={languagePickerRef}
-                      variant="ghost"
-                      color="body.base"
-                      transition="color 0.2s"
-                      px={{ base: "2", xl: "3" }}
-                      _hover={{
-                        color: "primary.hover",
-                        "& svg": {
-                          transform: "rotate(10deg)",
-                          transition: "transform 0.5s",
-                        },
-                      }}
-                      _active={{
-                        color: "primary.hover",
-                        bg: "primary.lowContrast",
-                      }}
-                      sx={{
-                        "& svg": {
-                          transform: "rotate(0deg)",
-                          transition: "transform 0.5s",
-                        },
-                      }}
-                    >
-                      <Icon
-                        as={BsTranslate}
-                        fontSize="2xl"
-                        verticalAlign="middle"
-                        me={2}
-                      />
-                      <Text hideBelow="lg" as="span">
-                        {t("common:languages")}&nbsp;
-                      </Text>
-                      {locale!.toUpperCase()}
-                    </MenuButton>
-                  </LanguagePicker>
-                </HStack>
-              )}
+              {/* avoid rendering desktop menu version on mobile */}
+              <Show above="md">
+                <Search {...searchModalDisclosure} />
+                <DesktopNavMenu toggleColorMode={toggleColorMode} />
+              </Show>
 
-              <MobileNavMenu
-                {...mobileNavProps}
-                linkSections={linkSections}
-                hideFrom="md"
-                toggleSearch={searchModalDisclosure.onOpen}
-                drawerContainerRef={navWrapperRef}
-              />
+              <Show below="md">
```

Comment:
<img width="769" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/0c154622-2ad5-4087-ab24-3cce15955e73">

Edge case at the exact `md` breakpoint width (768px) where both are showing up... seems `above` and `below` are both inclusive of the breakpoint.

Maybe this instead?

```suggestion
              <Hide sbove="md">
```

(If we do this, need to import `Hide` above)

Reply:
Ah good catch! thanks, will test the `Hide` component and see if we are not missing or adding anything new.